k8s/console/deployment/args.yaml

```diff
@@ -7,6 +7,7 @@ spec:
     spec:
       containers:
         - name: pomerium-console
+          args: ['serve']
```

Comment:
Seems unrelated?

Reply:
Sorry, some of these unrelated changes are a result of running the pre-commits. Specifically, cspell seems to always modify k8s files. I may need to talk to Denis to figure this out so it doesn't continue on future PRs.

Reply:
https://github.com/pomerium/documentation/pull/303

Reply:
@calebdoxsey didn't realize that was what was needed, will keep in mind going forward. Thanks!